scalafix/rules/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -132,13 +136,55 @@ case class Collectionstrawman_v0(index: SemanticdbIndex)
           close <- ctx.matchingParens.close(open.asInstanceOf[Token.LeftBracket])
         } yield
           ctx.replaceToken(open, "(") +
-            ctx.replaceToken(close, ")")).getOrElse(Patch.empty)
+            ctx.replaceToken(close, ")")
+        ).asPatch
+    }.asPatch
+
+  def replaceTupleZipped(ctx: RuleCtx) =
+    ctx.tree.collect {
+      case tupleZipped(Term.Select(Term.Tuple(args), name)) =>
+        val removeTokensPatch =
+          (for {
+            zipped <- name.tokens.headOption
+            closeTuple <- ctx.tokenList.leading(zipped).find(_.is[Token.RightParen])
+            openTuple <- ctx.matchingParens.open(closeTuple.asInstanceOf[Token.RightParen])
+            maybeDot = ctx.tokenList.slice(closeTuple, zipped).find(_.is[Token.Dot])
+          } yield {
+            ctx.removeToken(openTuple) +
+              maybeDot.map(ctx.removeToken).asPatch +
+              ctx.removeToken(zipped)
+          }).asPatch
+
+        def removeSurroundingWhiteSpaces(tk: Token) =
```

Comment:
Utilities for trimming whitespace are severely missing in scalafix. I'd love to hear if you have idea how this can be generalized

Reply:
it would definitely be handy to have such utility in the api. Maybe something like `ctx.trimLeft(token)` / `ctx.trimRight(token)` or `ctx.removeTokens(tokenList.spacesBefore(token))` / `ctx.removeTokens(tokenList.spacesAfter(token))`. I can put more thoughts on this and create a PR in scalafix with a proposal.